pages/transaction/[hash].tsx

```diff
@@ -229,37 +256,76 @@ const TransactionInfo = ({ data, loaded }) => {
         )}
       </Flex>
       <CardContainer>
-        {TRANSACTION_INFO_CARDS.map(card => (
-          <Card
-            key={card.key}
-            mb="1rem"
-            width={{
-              base: 'max(20rem, 100% - 0.5rem)',
-              md: 'max(20rem, 50% - 1rem)',
-              '2xl': 'max(20rem, 33.333333% - 1rem)',
-            }}
-            label={card.label}
-            value={card.value(data)}
-            icon={card.icon}
-            isLoading={!loaded}
-          />
-        ))}
+        {TRANSACTION_INFO_CARDS.map(card => {
+          const value = card.value(data)
+          if (value === null) return null
+          return (
```

Comment:
I'm not a big fun of inline if, but can we do 
```suggestion
          return value && (
```